README.md

```diff
@@ -75,6 +75,73 @@ The PMEM-CSI driver supports running in different modes, which can be controlled
 
 * **_Unified_** mode is intended to run the driver in a single host, mostly for testing the driver in a non-clustered environment.
 
+### Volume Persistency
+
+Normally, volumes are provided by a storage backend that is independent
+of a particular node. When a node goes offline, the volume can be
+mounted elsewhere. With PMEM-CSI, volumes are *local* and thus can
+only be used on the node where they were created.
+
+This limitation needs to be considered when designing and deploying
+applications that are to use local storage:
+
+* Persistent Volumes  
```

Comment:
Keeping in mind of local storage limitations, we better support only delayed binding `WaitForFirstConsumer`.

Reply:
You are looking at this from the perspective of an implementor ("let's use `WaitForFirstConsumer`"). Can we first discuss it in more abstract terms? Are you saying that "Persistent Volumes" aren't useful and that we should instead implement "Delayed Persistent Volumes"?